cypress/e2e/felony/class-a/pages/f-classa-main-1-ineligble.cy.ts

```diff
@@ -0,0 +1,15 @@
+import testIneligiblePage from "../../utils/testIneligiblePage"
+
+describe("Felony Classification Page", () => {
+  it('should redirect to Felony Class A page when choosing Felony Class A option', () => {
+    cy.visit('/calculator/head-felony-4-cont');
+    cy.get('[data-cy="calc-choice-0"]')
+      .should('be.visible')
+      .should('contain', 'Felony Class A')
+      .click();
+    cy.url().should('include', 'calculator/f-classa-main-1-ineligble');
```

Comment:
Hi, the slug is misspelled here and in the file title. Just missing an "i" in ineligible.

Reply:
@e-papanicolas Hi, the misspelled slug is what is live on our main site and Figma. I'll correct the spelling in the test and update Sanity and Figma for this change.